.github/workflows/build.yml

```diff
@@ -73,6 +55,7 @@ jobs:
     - name: Print hashes
       if: matrix.os == 'ubuntu-latest'
       run: |
+        pip install -U wheel
```

Comment:
Why manually installing `wheel` here?

Reply:
This was a left over from another branch. Thanks for catching this.